### DISCUSSION TITLE
Export payment history and import on new phone

### DISCUSSION THREAD

**Discussion by tonystark79** [ACINQ/phoenix#543] [Q&A] [answered]
Hello friends! 
As the title says, Is it possible to export the payment history on one device and import it back on a new device?
(I know I should not run two instances simultaneously, I´m changing my phone for a newer one)

**Comment by robbiehanson** [comment, marked as answer]
If your old & new phone are iOS, then your payment history will automatically sync to the new phone when you restore your wallet. To verify go to: Phoenix > Settings > Payments Backup, and ensure that "iCloud backup" is enabled.

If one of the phones in Android: I'm afraid this isn't implemented yet. Although you can export your payment history from the app.

**Reply by tonystark79** [reply]
Thank you! 
Didn´t know the iOS version would sync...

Sadly, in my case both phones are Android and I´ve found how to export from the older one (which I already did)...

But is there a way to import it into the new phone?

**Reply by robbiehanson** [reply]
> But is there a way to import it into the new phone?

No, I'm afraid that's not currently implemented. Sorry about that.

**Reply by tonystark79** [reply]
Thanks my friend!!!! :))
